Se agregaron los tests de Filters y otros

elyansan opened the pull request:
Se hicieron tests de Filters, Multiselect, MultiselectOptions y ProgressBar (Luis Manuel Gallegos) con el 100% de coverage.

elyansan committed:
Se agregaron los tests de Filters y otros
Se hicieron tests de Filters, Multiselect, MultiselectOptions y
ProgressBar (Luis Manuel Gallegos) con el 100% de coverage.

Djmr5 reviewed (changes requested):
Fallan 11 tests xd, arreglar porfa :)
![image](https://github.com/TC3005B-562-2024/React-Frontend/assets/85468901/93dbc82a-5441-4c17-a365-b1fc116f2b55)

elyansan committed:
Se corrigen los tests de Filters
Se aprueban todos los tests correspondientes a Filters, Multiselect y
MultiselectOptions.
